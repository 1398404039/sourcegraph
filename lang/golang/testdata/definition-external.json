[
  {
    "id": "0",
    "jsonrpc": "2.0",
    "method": "initialize",
    "params": {
      "rootPath": "testdata/promexample",
      "capabilities": {}
    }
  },
  {
    "id": "1",
    "jsonrpc": "2.0",
    "method": "textDocument/definition",
    "params": {
      "textDocument": {
        "uri": "file:///main.go"
      },
      "position": {
        "line": 9,
        "character": 55
      }
    }
  },
  {
    "id": "2",
    "jsonrpc": "2.0",
    "method": "shutdown",
    "params": null
  }
]
