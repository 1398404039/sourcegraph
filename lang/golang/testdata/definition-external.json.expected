[
  {
    "result": {
      "capabilities": {
        "hoverProvider": true,
        "definitionProvider": true,
        "referencesProvider": true
      }
    },
    "id": "0",
    "jsonrpc": ""
  },
  {
    "result": [
      {
        "uri": "golang://github.com/prometheus/client_golang/prometheus.CounterOpts"
      }
    ],
    "id": "1",
    "jsonrpc": ""
  },
  {
    "result": true,
    "id": "2",
    "jsonrpc": ""
  }
]